content/blog/salesforce-vs-in-house-custom-development.md

```diff
@@ -0,0 +1,109 @@
+
+- We understood that using software to streamline workflows needs more than small tweaks and customizations
+- It needs the experience and investigative understanding of a truly cross-functional team to discover the nuances and interdependencies of a complex system
+- The solution Incubyte needed to build had to be reliable, fast, effective, and affordable. Most importantly, it needed to enable Genie's agents to enjoy their tasks and do better daily
+
+## Progress in two years: Salesforce vs Custom Software Solution
+
+| Salesforce | Metamorphosis |
+| --- | --- |
+| Needed a lot of upfront planning before onboarding any users | Genie increased the number of users by slowly and steadily improving the system iteratively and consistently. |
+| Agencies specializing in niche areas had to be engaged since the low-code implementation still required professional services in MuleSoft, Cloud Data Migration, etc. | The cross-functional team at Incubyte owned the entire stack from discovery to deployment, reducing communication overhead and dependencies across teams |
+| ~100 users onboarded | 800+ users onboarded. |
+
+## How we did it
+
+- We started with small solutions for specific workflows, and built on top of them
+- The Incubyte team worked with agility and discipline.
+- We released small upgrades, got feedback from end users, and iterated - ensuring that the users got the most impactful features fast and could use them effectively
+- We automated everything from day one, thanks to TDD and CI/CD - the build process was clearly defined and scripted, making it consistent and repeatable. This helped us improve our release frequency and lead time to release. It made our end users really happy as they saw the features from their wishlists in the product way sooner than expected
+- Testing and quality checks were automated, keeping manual testing to a minimum
+
+_Metamorphosis was always release-ready._
+
+## Graphic: Scoreboard
+
+| Salesforce Scoreboard | Metamorphosis Scoreboard |
+| --- | --- |
+| Users Onboarded - 100 | Users Onboarded - 800+ |
+| Operational CTC for 100 users = 270,000$ | Operational CTC per year for 800+ users = 100,000$ |
+| Typical Lead Time to Release = Undefined | Typical Lead Time to Release = 2 Weeks |
+
+## The Impact
+
+Genie started saving 4000 hours a day across 800 users.
+
+## What Incubyte Achieved
+
+- **Saved 4000 hours a day.** Team members onboarded by the Metamorphosis project have a seamless workflow that resulted in time saving
+- **Over 800 users onboarded** by maintaining smaller, stable, consistent releases
+- **It's a tailored solution**, it is more than customization, because Incubyte developed a solution with the end user and their use cases in mind.
+- **By being fast, economical, and agile**, within two years, Metamorphosis added value to the company 
```

Comment:
This line does not sound impactful. Can we change it to something like, in a short span of 2 yrs, Metamorphis was able to add value to the company?